Build fail for projects generated with archetype 25 when using AEM SDK version >= 5026

Creating a project with archetype 25 and `cloud` SDK leads to build failures:

```
[INFO] Analyzing feature 'com.adobe.aem.guides.wknd:aem-guides-wknd.analyse:slingosgifeature:aggregated-author.prod:0.0.1-SNAPSHOT' finished : 0 warnings, 1 errors.
[ERROR] repoinit: Parsing error in repoinit from extension : Encountered "" at line 15, column 37.
Was expecting one of:

[ERROR] Analyser detected errors on feature 'com.adobe.aem.guides.wknd:aem-guides-wknd.analyse:slingosgifeature:aggregated-author.prod:0.0.1-SNAPSHOT'. See log output for error messages.
```

--- a/src/main/archetype/pom.xml
+++ b/src/main/archetype/pom.xml
@@ -98,7 +98,7 @@
 #end
 #if ( $aemVersion == "cloud")
         <aem.sdk.api>SDK_VERSION</aem.sdk.api>
-        <aemanalyser.version>0.9.0</aemanalyser.version>
+        <aemanalyser.version>0.9.2</aemanalyser.version>
 #end
         <componentGroupName>$appTitle</componentGroupName>
     </properties>

